frontend/src/components/gotEnzymes/Landing.vue

```diff
@@ -99,24 +97,28 @@
               <hr class="mt-6" />
               <h4 id="how-it-was-made" class="is-info is-size-4">How it was made</h4>
               <p>
-                Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a
-                piece of classical Latin literature from 45 BC, making it over 2000 years old.
-                Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked
-                up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and
-                going through the cites of the word in classical literature, discovered the
-                undoubtable source. Lorem Ipsum comes from sections 1.10.32 and 1.10.33 of "de
-                Finibus Bonorum et Malorum" (The Extremes of Good and Evil) by Cicero, written in 45
-                BC. This book is a treatise on the theory of ethics, very popular during the
-                Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes
-                from a line in section 1.10.32.
+                Data in this database are predicted from available advanced deep learning tools:
+                k<sub>cat</sub> published in <i>Nature Catalysis</i> in 2022 (<a
+                  href="https://doi.org/10.1038/s41929-022-00798-z"
+                  >doi:10.1038/s41929-022-00798-z</a
+                >). Our vision for this database is to store every possible predicted parameter for
+                any available enzyme to facilitate computational applications such as
+                flux simulations, and for use in metabolic engineering.
               </p>
 
               <hr class="mt-6" />
               <h4 id="citation" class="is-info is-size-4">Citation</h4>
               <p>
-                To cite GotEnzymes please use this [link]. For more information about the prediction
-                methods, have a look at reference title [link] for kcat prediction and KM
-                prediction.
+                <i>Manuscript under consideration</i>. For more information about the k<sub>cat</sub
+                >
```

Comment:
I fee it's a bit duplicated to use this publication again since it has been referred in the previous paragraph already. Moreover, this section is about `citation` and I guess it should be the manuscript to NAR that should be cited for GotEnzymes, right?

Reply:
Let's see where this gets published first, and then we can remove the duplicated DLKcat reference here.